docs/reference/settings.md

```diff
@@ -1495,39 +1495,6 @@ packages.
 
 ---
 
-#### [`allow-insecure-host`](#pip_allow-insecure-host) {: #pip_allow-insecure-host }
```

Comment:
Is this supposed to be gone? I think this is breaking

Reply:
Good catch